schema.yml

```diff
@@ -653,78 +644,78 @@ paths:
           schema:
             type: string
       responses:
-        '200':
+        "200":
           description: Successful response with deployment details.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/TickResponseBody'
-        '500':
+                $ref: "#/components/schemas/TickResponseBody"
+        "500":
           description: Internal server error.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ErrorResponseBody'
+                $ref: "#/components/schemas/ErrorResponseBody"
 
-  /api/v2/src101/{deploy_hash}/address/{address_btc}:
-    get:
-      summary: Get tokenid of SRC-101 by address_btc
-      description: Retrieve deployment details for a specific deploy_hash in the deploy_hash API.
-      parameters:
-        - in: query
-          name: limit
-          schema:
-            type: integer
-            example: 2
-            minimum: 1
-            default: 50
-          description: The maximum number of transactions to retrieve per page
-        - in: query
-          name: page
-          schema:
-            type: integer
-            minimum: 1
-            default: 1
-          description: The page number of transactions to retrieve
-        - in: query
-          name: sort
-          schema:
-            type: string
-            default: DESC
-            enum: [ASC, DESC]
-          description: Sort order [ASC, DESC]
-        - in: query
-          name: prim
-          required: true
-          example: "true"
-          schema:
-            type: string
-        - in: path
-          name: deploy_hash
-          required: true
-          example: "8d40a2b97c0fa638fa18daf564f916260a22e7f2eca9a786763f2eeaab57541d"
-          description: The hash of the deploy transaction.
-          schema:
-            type: string
-        - in: path
-          name: address_btc
-          required: true
-          example: "miuyV1UoMA25QE8heKjWX8A7RtAn26tLYu"
-          schema:
-            type: string
-        responses:
-        '200':
-          description: Successful response with deployment details.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/TokenidSrc101ResponseBody'
-        '500':
-          description: Internal server error.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorResponseBody'
+  # /api/v2/src101/{deploy_hash}/address/{address_btc}:
```

Comment:
these tables haven't fully populated in the db yet until likely tomorrow, but this is a critical query to get in for this week